test/acse-conf/acse-test.json

```diff
@@ -9,10 +9,6 @@
       "location": "eastus"
     },
     {
-      "cluster_definition": "swarm.json",
```

Comment:
we will need to keep testing swarm, until all headers are gone.  is the "swarm.json" using an older header?